packages/astro-rss/README.md

```diff
@@ -202,6 +185,112 @@ export const get = () => rss({
+
+An example feed item might look like:
+
+```js
+const item = {
+  title: "Alpha Centauri: so close you can touch it",
+  link: "/blog/alpha-centuari",
+  pubDate: new Date("2023-06-04"),
+  description: "Alpha Centauri is a triple star system, containing Proxima Centauri, the closest star to our sun at only 4.24 light-years away.",
+  categories: ["stars", "space"]
+}
+```
+
+### `title`
+
+Type: `string (required)`
+
+The `<title>` attribute of the item in the feed.
+
+### `link`
+
+Type: `string (required)`
+
+The `<link>` attribute of the item in the feed containing the URL of the item on the web.
+
+### `pubDate`
+
+Type: `Date (required)`
+
+Indicates when the item was published.
+
+### `description`
+
+Type: `string (optional)`
+
+If the item is complete within itself, that is you are publishing the full content of the item in the feed, the `description` field may contain the full text (entity-encoded HTML is permitted). If the item is a stub, then the `description` may contain a synopsis of the item.
+
+### `content`
+
+Type: `string (optional)`
+
+If you want to supply both a short description and also the full content in an item, set the `content` field to the full, encoded text. See the [recommendations from the RSS spec for how to use and differentiate between `description` and `content`](https://www.rssboard.org/rss-profile#namespace-elements-content-encoded).
+
+### `categories`
+
+Type: `string[] (optional)`
+
+If you use tags or categories to categorize your content, you can add them as the `categories` field. They will be output as multiple `<category>` elements.
+
+### `author`
+
+Type: `string (optional)`
+
+Useful for multi-author blogs, the `author` field provides the email address of the person who wrote the item.
+
+### `commentsUrl`
+
+Type: `string (optional)`
+
+The `commentsUrl` defines a URL of a web page that contains comments on the item.
+
+### `source`
+
+Type: `object (optional)`
+
+Items that are republished from other publications may define a `source` which defines the `title` and `url` of the original feed in which it was published.
+
+#### `title`
+
+Type: `string (required)`
+
+If you define a `source` you must define that source's `title`. It is the name of the original feed in which the item was published.
+
+#### `url`
+
+Type: `string (required)`
+
+If you define a `source` you must also define that source's `url` which identifies the URL of the original feed in which the item was published.
+
+### `enclosure`
+
+Type: `object (optional)`
+
+Items that include media as part of the feed, like a podcast, can define an `enclosure` which is made of three required fields, a `url`, `length`, and `type`.
```

Comment:
``````suggestion
An object to specify properties for an included media source (e.g. a podcast) with three required values: `url`, `length`, and `type`.

```js
const item = {
	title: "Alpha Centauri: so close you can touch it",
  link: "/blog/alpha-centuari",
  pubDate: new Date("2023-06-04"),
	enclosure:
		{
			url: "https://mydomain/media/alpha-centauri.aac",
			length: 124568,
			type: "audio/aac"
		}
}
```
``````
I also think an example would be helpful here! Note, if the URL is from your own site, do you still need to use a full URL here? This is just an example. Adjust to make this correct!

Reply:
The RSS spec says that you do need this to be a full URL. However, we can treat this like `link`, check if it's a valid URL and if it isn't run it through `createCanonicalURL` to make a full URL for a relative path.